### DISCUSSION TITLE
doris 集群的稳定性、和数据同步链接的异常

### DISCUSSION THREAD

**Discussion by 936447225** [apache/doris#24208] [A - General / Q&A]
1、doris 经常发生tablet版本重平衡会报出以下异常，导致任务取消
![image](https://github.com/apache/doris/assets/42171091/23adf720-a61b-4cb5-8aa8-579636e9eb61)
2、jdbc链接偶尔出现：RpcException, msg: java.net.ConnectException: finishConnect(..) failed: Connection refused异常
![image](https://github.com/apache/doris/assets/42171091/1b28435f-fc27-46e4-9edd-b453aa89e972)
3、通过stream load json文件同步doris 出现
[PUBLISH_TIMEOUT]transaction commit successfully, BUT data will be visible late

**Comment by yagagagaga** [comment]
What is your version of Doris?
